articles/tokens/refresh-token.md

```diff
@@ -1,152 +1,173 @@
 ---
-description: A refresh token allows an application to request Auth0 to issue a new id_token directly, without needing to re-authenticate the user.
+title: Refresh Token
+description: What is a Refresh Token and how you can use it.
 ---
 
-# Refresh Tokens
+# Refresh Token
```

Comment:
The corollary of this is that SPA cannot, under no conditions, use refresh token. SPA should use silentAuth to refresh session/access.